express/code/features/table-of-contents-seo/table-of-contents-seo.js

```diff
@@ -433,95 +30,185 @@ function buildMetadataConfigObject() {
 
 export default async function setTOCSEO() {
   ({ createTag, getMetadata } = await import(`${getLibs()}/utils/utils.js`));
-  const doc = document.querySelector('main');
   const config = buildMetadataConfigObject();
 
-  try {
-    const { getGnavHeight } = await import(`${getLibs()}/blocks/global-navigation/utilities/utilities.js`);
-    MOBILE_NAV_HEIGHT = getGnavHeight();
-  } catch (e) {
-    window.lana?.log(`Error getting gnav height ${e}`);
-  }
-
-  const desktopSkipLink = createTag('a', {
-    href: '#desktop-toc',
-    class: 'skip-link desktop-skip-link',
-    'aria-label': 'Skip to Table of Contents',
+  const toc = createTag('div', {
+    class: 'toc toc-container',
+    role: 'navigation',
+    'aria-label': 'Table of Contents',
   });
-  desktopSkipLink.textContent = 'Skip to Table of Contents';
 
-  const mobileSkipLink = createTag('a', {
-    href: '#mobile-toc-button',
-    class: 'skip-link mobile-skip-link',
-    'aria-label': 'Skip to Mobile Table of Contents',
+  const title = createTag('div', {
```

Comment:
Is there a reason this can't be an actual <button> if we're just creating an el?